pkg/resources/deployment.go

```diff
@@ -84,9 +86,19 @@ func newPodTemplateSpec(hawtio *hawtiov1alpha1.Hawtio, isOpenShift4 bool, openSh
 	}
 	volumes := newVolumes(hawtio, isOpenShift4)
 
+	labels := labelsForHawtio(hawtio.Name)
+	selector := strings.ReplaceAll(buildVariables.AdditionalLabels, ";", ",")
```

Comment:
Why not mandating the labels to be comma separated?

Reply:
I wanted to keep it the same with https://github.com/apache/camel-k/pull/2646 but I saw you've raised similar suggestion there. I've changed the changes.